tests/api/utils/projects/test_mlrun_consumer.py

```diff
@@ -0,0 +1,206 @@
+import datetime
+
+import pytest
+import sqlalchemy.orm
+
+import mlrun.api.schemas
+import mlrun.api.utils.projects.consumers.base
+import mlrun.api.utils.projects.consumers.mlrun
+import mlrun.api.utils.projects.manager
+import mlrun.api.utils.singletons.db
+import mlrun.config
+import mlrun.errors
+
+
+@pytest.fixture()
+async def mlrun_consumer() -> mlrun.api.utils.projects.consumers.mlrun.Consumer:
+    mlrun_consumer = mlrun.api.utils.projects.consumers.mlrun.Consumer()
+    return mlrun_consumer
```

Comment:
Why 2 lines?
Here (and in some other places), no need to alias/assign to var if there isn't a reuse or if the expression is not really bulky ("alias" only where helps readability).
This is a "weak" style comment though, so, take it or leave it